x-pack/package.json

```diff
@@ -46,27 +47,44 @@
     "@types/d3-time": "^1.0.7",
     "@types/d3-time-format": "^2.1.0",
     "@types/elasticsearch": "^5.0.30",
+    "@types/file-type": "^5.2.1",
```

Comment:
The latest version of `file-type` includes its own type definitions and doesn't have any breaking API changes that should impact our usage.